big_o_exercise/readme.md

```diff
@@ -44,6 +79,8 @@ function logAtLeast10(n) {
         console.log(i);
     }
 }
+Time: O(n) if n > 10, O(1) if n < 10
```

Comment:
since big O notation only cares about sufficiently large values of n, it's fine to just say O(n) here.